code-of-conduct.md

```diff
@@ -0,0 +1,23 @@
+# Code of Conduct
+
+## Introduction
+
+Biomakespace is building a multidisciplinary, volunteer-driven community laboratory for collaborative experimentation at the interface of biology and engineering. We aim to provide a supportive, safe environment for discovery and innovation through team-based projects and curiosity-driven research.
+
+## Scope
+
+This code of conduct applies to all Biomakespace interactions and spaces, both online and in person. By attending Biomakespace events, space and posting in our online forums, you are agreeing to this code of conduct.
+
```

Comment:
Need to add section on revising the code of conduct.

Reply:
💡